cypress/e2e/smoke/content/cms-page-dms.cy.ts

```diff
@@ -0,0 +1,48 @@
+import { faker } from '@faker-js/faker';
+import { container } from '@utils';
+import { ContentStaticFixtures } from '@interfaces/smoke';
+import { CreateCmsPageScenario, CreateStoreScenario, UserLoginScenario } from '@scenarios/backoffice';
+import { SelectStoreScenario } from '@scenarios/yves';
+
+(Cypress.env('isDynamicStoreEnabled') ? describe : describe.skip)(
+  'cms page assignment to store',
+  { tags: '@smoke' },
+  () => {
+    const userLoginScenario = container.get(UserLoginScenario);
+    const createStoreScenario = container.get(CreateStoreScenario);
+    const createCmsPageScenario = container.get(CreateCmsPageScenario);
+    const selectStoreScenario = container.get(SelectStoreScenario);
+
+    let staticFixtures: ContentStaticFixtures;
+
+    before((): void => {
+      staticFixtures = Cypress.env('staticFixtures');
+      staticFixtures.store.name = staticFixtures.store.name + '_' + faker.string.alpha({ casing: 'upper' });
+      staticFixtures.cmsPageName = staticFixtures.cmsPageName + '_' + faker.string.alpha();
```

Comment:
Do you have such a requirement to create a new store every test? I expect the new store to be created only once and then all tests should use it.